tron/serialize/runstate/statemanager.py

```diff
@@ -161,7 +163,11 @@ def restore(self, job_names, skip_validation=False):
                 for job_name, job_state in jobs.items()
             }
             for result in concurrent.futures.as_completed(results):
-                jobs[results[result]]["runs"] = result.result()
+                try:
+                    jobs[results[result]]["runs"] = result.result()
+                except Exception:
+                    log.exception(f"Unable to restore state for {results[result]} - exiting to avoid corrupting data.")
+                    sys.exit(ExitCode.fail)
```

Comment:
sorry, just noticed this ExitCode import - i'd probably just do a `sys.exit(1)` since cmd_utils is ostensibly meant for CLI utils like tronctl/tronview/etc

Reply:
Np, thank youu